doc/snippets/Microsoft.Data.SqlClient/SqlBatch.xml

```diff
@@ -1,285 +1,440 @@
-<?xml version="1.0"?>
-<docs>
+﻿<docs>
   <members name="SqlBatch">
     <SqlBatch>
-      <summary></summary>
-      <remarks>
-        <format type="text/markdown">
+      <example>
+        The following example creates a <see cref="Microsoft.Data.SqlClient.SqlConnection" /> and a <see cref="Microsoft.Data.SqlClient.SqlBatch" />, then adds multiple <see cref="Microsoft.Data.SqlClient.SqlBatchCommand" /> objects to the batch. It then executes the batch, creating a <see cref="Microsoft.Data.SqlClient.SqlDataReader" />. The example reads through the results of the batch commands, writing them to the console. Finally, the example closes the <see cref="Microsoft.Data.SqlClient.SqlDataReader" /> and then the <see cref="Microsoft.Data.SqlClient.SqlConnection" /> as the <c>using</c> blocks fall out of scope.
```

Comment:
You can wrap the text inside a <para> tag and move the example tag to the code part.

Reply:
I think I need a better explanation of what you're suggesting here... Based on my understanding, the example tag should be the top level and can contain more than just the code itself, ideally some text explaining what's happening in the example. Either way, this seems to work, so I need a bit better understanding of what you're suggesting.

Reply:
something similar to:
```xml
<para>
The following example creates a <see cref="Microsoft.Data.SqlClient.SqlConnection" /> and a 
<see cref="Microsoft.Data.SqlClient.SqlBatch" />, then adds multiple <see cref="Microsoft.Data.SqlClient.SqlBatchCommand" />
 objects to the batch. It then executes the batch, creating a <see cref="Microsoft.Data.SqlClient.SqlDataReader" />. The example reads through the results of the batch commands, writing them to the console. Finally, the example closes the <see cref="Microsoft.Data.SqlClient.SqlDataReader" /> and then the <see cref="Microsoft.Data.SqlClient.SqlConnection" /> as the <c>using</c> blocks fall out of scope.
</para>
<example>
the code goes in here
</example>
```

Reply:
or basically you can move the para section inside the example tag

Reply:
Gotcha. I'm pretty sure the <example> needs to be the top level element and within the example you can have paragraphs or code.